### ISSUE TITLE
Aldynes (on SuperGrafx) crashes during the DEMO/Intro sequence

### ISSUE THREAD

**Issue by mathieulh** [agg23/openfpga-pcengine#22]
### Version (or build number)

0.3.0

### Steps to reproduce

Aldynes (on SuperGrafx) crashes during the DEMO/Intro sequence

Just let the demo run (without pressing run/start) and eventually (early in the demo) the game will crash.

### Expected Behavior

The game shouldn't crash

### Actual Behavior

The game crashes early in the demo/attract gameplay.

### Additional Context

The game doesn't crash on the TG16 MiSTer core or real hardware.

**Comment by agg23** [comment]
That's pretty surprising. I can confirm it's occurring for me as well.

Just to make sure, you did test this on MiSTer? I have no idea what would be causing a discrepancy between the cores.

**Comment by mathieulh** [comment]
Indeed, I have tested this game on MiSTer FPGA and it runs great, the demo/attract doesn't crash.
